src/simulator/src/setup.ts

```
@@ -0,0 +1,178 @@
+import { getToken } from '#/pages/simulatorHandler.vue';
+
+/**
+ * to resize window and setup things it
+ * sets up new width for the canvas variables.
+ * Also redraws the grid.
+ * @category setup
+ */
+export function resetup(): void {
+    const DPR = window.devicePixelRatio || 1;
+    if (lightMode) {
+        DPR = 1;
```

Biome (1.9.4)

[error] Can't assign DPR because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)